components/RootHomePage/Blog.js

```diff
@@ -97,7 +97,7 @@ const BlogTitle = styled.h4`
 
 const Blog = ({ className }) => {
   let markdownContent = '';
-  const currentBlogSlug = 'april-2020-update';
+  const currentBlogSlug = 'refactoring-THAT-conference-2020';
```

Comment:
Why is this hard-coded and not taking the latest blog post?

Reply:
@brettski  it was hard-coded originally and I wasn't aware there was a mechanism for getting the latest blog post.  Can you point me in the direction of that?

Reply:
@kburnell Yeah you're correct. So this is now yet another place to update when we add a new blog post. It should be added to [CONTRIBUTING.MD](https://github.com/ThatConference/that-website/blob/master/CONTRIBUTING.MD) page in the add blog section so it isn't forgotten in the future.